frontend/components/page/PageBreadcrumbs.vue

```diff
@@ -96,13 +96,9 @@ const eventStore = useEventStore();
 let organization: Organization;
 let group: Group;
 let event: Event;
+const organizationRegex = /^\/organizations\/[0-9a-fA-F-]+(\/about)?$/;
```

Comment:
Shouldn't we do the same thing for events?

Reply:
Yeah, my teammate is working on that, however, we just wanted to check if we're on the right track.